foreman/foreman.spec

```diff
@@ -621,6 +641,10 @@ if [ ! -e %{_datadir}/%{name}/config/initializers/encryption_key.rb -a \
   ln -s %{_sysconfdir}/%{name}/encryption_key.rb %{_datadir}/%{name}/config/initializers/
 fi
 
+# copy pregenerated apipie cache files
```

Comment:
This is mainly for case of Foreman update. This code is responsible for copying of the plugin's pre-built caches to the main `apipie-cache` directory. It can not happen during the build time because of conflict (resource with same name in core and plugin).